alf/utils/common.py

```diff
@@ -441,7 +445,10 @@ def forward(self, observation):
         """Normalize a given observation. If during unroll, then first update
         the normalizer. The normalizer won't be updated in other circumstances.
         """
-        if not is_training():
+        # TODO: To update on unroll, we need to fix the case where test exp
+        # should be excluded.
```

Comment:
test is performed using policy_trainer._step(), which is under eval mode. So there seems no problem.

Reply:
Done.